src/main/java/pl/javastart/task/logic/TicketShop.java

```diff
@@ -0,0 +1,91 @@
+package pl.javastart.task.logic;
+
+import pl.javastart.task.model.*;
+
+import java.util.Scanner;
+
+public class TicketShop {
+    private static final double SHIPPING_COST = 5;
+    private static final String ONLINE_TICKET = "online";
+    private static final String STANDARD_TICKET = "standard";
+    private static final String GIFT_TICKET = "gift";
+
+    private static double countTicketFinalPrice(Ticket ticket) {
+        String ticketType = ticket.getType();
+        return switch (ticketType) {
+            case ONLINE_TICKET -> ticket.getTotalPrice();
+            case STANDARD_TICKET -> ticket.getTotalPrice() + SHIPPING_COST;
+            case GIFT_TICKET -> ticket.getTotalPrice() + SHIPPING_COST + (ticket.getDiscount() * ticket.getTotalPrice());
```

Comment:
Tutaj wygląda to tak, jakbyś dwa razy naliczała zniżkę, ale nie bierzesz pod uwagę 5% ceny z treści zadania
`cena jest taka jak biletu standard, powiększona o dodatkowe 5% ceny podstawowej ze zniżką`, czyli powinno się tutaj pojawić `ticket.getTotalPrice() * 5%`

Reply:
Byłam absolutnie przekonana, że to jest ta sama zniżka, którą wykorzystujemy do obliczania ceny po zniżce (a to tylko zbieżność cyferek) :)